src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+        }),
+        (n._handleSwipe = function () {
+          var t = Math.abs(this.touchDeltaX);
+          if (!(t <= 40)) {
+            var e = t / this.touchDeltaX;
+            (this.touchDeltaX = 0), e > 0 && this.prev(), e < 0 && this.next();
+          }
+        }),
+        (n._addEventListeners = function () {
+          var t = this;
+          this._config.keyboard &&
+            e(this._element).on("keydown.bs.carousel", function (e) {
+              return t._keydown(e);
+            }),
+            "hover" === this._config.pause &&
+              e(this._element)
+                .on("mouseenter.bs.carousel", function (e) {
+                  return t.pause(e);
+                })
+                .on("mouseleave.bs.carousel", function (e) {
+                  return t.cycle(e);
+                }),
+            this._config.touch && this._addTouchEventListeners();
+        }),
+        (n._addTouchEventListeners = function () {
+          var t = this;
+          if (this._touchSupported) {
+            var n = function (e) {
+                t._pointerEvent && b[e.originalEvent.pointerType.toUpperCase()]
+                  ? (t.touchStartX = e.originalEvent.clientX)
+                  : t._pointerEvent ||
+                    (t.touchStartX = e.originalEvent.touches[0].clientX);
+              },
+              i = function (e) {
+                t._pointerEvent &&
+                  b[e.originalEvent.pointerType.toUpperCase()] &&
+                  (t.touchDeltaX = e.originalEvent.clientX - t.touchStartX),
+                  t._handleSwipe(),
+                  "hover" === t._config.pause &&
+                    (t.pause(),
+                    t.touchTimeout && clearTimeout(t.touchTimeout),
+                    (t.touchTimeout = setTimeout(function (e) {
+                      return t.cycle(e);
+                    }, 500 + t._config.interval)));
+              };
+            e(this._element.querySelectorAll(".carousel-item img")).on(
+              "dragstart.bs.carousel",
+              function (t) {
+                return t.preventDefault();
+              }
+            ),
+              this._pointerEvent
+                ? (e(this._element).on("pointerdown.bs.carousel", function (t) {
+                    return n(t);
+                  }),
+                  e(this._element).on("pointerup.bs.carousel", function (t) {
+                    return i(t);
+                  }),
+                  this._element.classList.add("pointer-event"))
+                : (e(this._element).on("touchstart.bs.carousel", function (t) {
+                    return n(t);
+                  }),
+                  e(this._element).on("touchmove.bs.carousel", function (e) {
+                    return (function (e) {
+                      e.originalEvent.touches &&
+                      e.originalEvent.touches.length > 1
+                        ? (t.touchDeltaX = 0)
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)